frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -22,6 +22,7 @@ import { PersistentVolumeClaimModel } from '../models';
 
 const { common, ExpandPVC } = Kebab.factory;
 const menuActions = [
+  ...Kebab.getExtensionsActionsForOCS(PersistentVolumeClaimModel),
```

Comment:
So If I am not mistaken, this is completely based on `PVCModel` so it will get activated if the ceph-plugin is loaded. I think we need to add a feature flag filter or associate a required flag with your extension point.

Reply:
`isOCSKebabActions` is already taking care of it. Based on this check, it will be enabled or disabled.

Reply:
How is `isOCSKebabActions` checking for feature flag? If I am not mistaken no flag is associated with the extension.

Reply:
This feature will be enabled for extension, once [PR](https://github.com/openshift/console/pull/3383/files) gets merged. 
I will send out a separate PR to address this.

Reply:
Does k8's natively support cloning or is it strictly an OCS feature?

Reply:
This will be enabled only when OCS operator is installed.